certbot/certbot/_internal/main.py

```diff
@@ -1233,19 +1233,11 @@ def renew_cert(config, plugins, lineage):
 
     renewed_lineage = _get_and_save_cert(le_client, config, lineage=lineage)
 
-    notify = zope.component.getUtility(interfaces.IDisplay).notification
-    if installer is None:
-        notify("new certificate deployed without reload, fullchain is {0}".format(
-               lineage.fullchain), pause=False)
-    else:
+    if installer and not config.dry_run:
         # In case of a renewal, reload server to pick up new certificate.
-        # In principle we could have a configuration option to inhibit this
-        # from happening.
-        # Run deployer
         updater.run_renewal_deployer(config, renewed_lineage, installer)
+        display_util.notify(f"Restarting {config.installer} server after certificate renewal")
```

Comment:
Ok now I'm torn on the reload vs restart wording. Because we (first) do a "graceful restart" for apache and a "reload" for nginx, since neither actually stops the server. And I wouldn't want people to be worried that it's actually stopping and starting the whole thing. But also this is technically just a generic function that we call, but also the most common cases are apache and then nginx. What do you think?

Reply:
Yes, at least without interrogating the installer for "graceful reload" support, I think "Reloading" is a better thing to say by default. Added it.